src/style.css

```diff
@@ -0,0 +1,55 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  width: 0;
+  height: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.container {
+  position: absolute;
```

Comment:
why do you need position: absolute? 
it will work without it if you remove redundant styles from html, body.